src/box/lua/misc.cc

```diff
@@ -400,13 +400,7 @@ lbox_tuple_format_new(struct lua_State *L)
 		lua_gettable(L, -2);
 		assert(! lua_isnil(L, -1));
 		const char *name = lua_tolstring(L, -1, &len);
-		fields[i].name = (char *)region_alloc(region, len + 1);
-		if (fields == NULL) {
-			diag_set(OutOfMemory, size, "region_alloc",
-				 "fields[i].name");
-			region_truncate(region, region_svp);
-			return luaT_error(L);
-		}
+		fields[i].name = (char *)xregion_alloc(region, len + 1);
```

Comment:
Please also replace `region_alloc_array` with `xregion_alloc_array` at line 376 while you are at it.

Reply:
Fixed. However, I added an explicit cast for the return value of xregion_alloc_array() because otherwise a compilation error occurs. I think it's because it's a cc file. There were no errors for region_alloc_array().